presto-native-execution/presto_cpp/main/TaskManager.cpp

```diff
@@ -47,15 +47,15 @@ namespace {
 // the file system and sets the path to it to the Task.
 static void maybeSetupTaskSpillDirectory(
     const core::PlanFragment& planFragment,
-    exec::Task& execTask) {
-  const auto baseSpillPath = SystemConfig::instance()->spillerSpillPath();
+    exec::Task& execTask,
+    const std::string& baseSpillPath) {
```

Comment:
It looks to me that this change is to set and pass the baseSpillPath to maybeSetupTaskSpillDirectory() instead of getting it in maybeSetupTaskSpillDirectory() impl. If so, how does it make difference? Why this change?